src/content/docs/apis/nerdgraph/get-started/nerdgraph-explorer.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Use the NerdGraph explorer
+tags:
+  - APIs
+  - NerdGraph
+  - Get started
+metaDescription: How to use the New Relic NerdGraph API explorer. 
+redirects:
+---
+
+import createAccount from 'images/create-account.png'
+
+import graphqlDocumentation from 'images/graphql-documentation.png'
+
+import mutationExample from 'images/mutation-example.png'
+
+import toolsMenu from 'images/tools-menu.png'
+
+NerdGraph is our main API for querying data and making configurations to New Relic features. One popular way to use NerdGraph and make API requests is with our NerdGraph explorer. 
+
+## Find the explorer [#find]
+
+To find the explorer: 
+
+* If your New Relic account uses an [EU data center](/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), go to [api.eu.newrelic.com/graphiql](https://api.eu.newrelic.com/graphiql).
```

Comment:
@zuluecho9 I'm not a big fan of splitting up sentences across bullets. You might consider just make a single sentence: If your New Relic account uses an [EU data center](https://docswebsitedevelop-devsitemigratebringovernerdgra.gtsb.io/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), go to [api.eu.newrelic.com/graphiql](https://api.eu.newrelic.com/graphiql);
otherwise use [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql).

If you still wanted to use bullets, maybe something like this:

Use the explorer for your region:

* Inside the EU: If your account uses an [EU data center](https://docswebsitedevelop-devsitemigratebringovernerdgra.gtsb.io/docs/using-new-relic/welcome-new-relic/get-started/our-eu-us-region-data-centers), go to [api.eu.newrelic.com/graphiql](https://api.eu.newrelic.com/graphiql).
* Outside the EU: If your account is outside the EU, use [api.newrelic.com/graphiql](https://api.newrelic.com/graphiql).